src/base/static/components/atoms/typography.js

```diff
@@ -108,6 +108,19 @@ Header6.propTypes = {
   classes: PropTypes.string,
 };
 
+const PanelLabel = ({ children, ...props }) => {
```

Comment:
I don't see the point of having hyper-specific components like PanelLabel in our atoms. And allowing the consumer of this atom to directly manipulate the styles by overriding the class name is an anti-pattern, as mentioned in my previous comment. We should avoid that pattern moving forward, and use CSS-in-JS for all new code that we write.

I would recommend looking up a resource like "developing a design language". I still have to learn about this myself, but I think this should be something like SecondaryText, TertiaryText, or similar. Then we can normalize the text throughout the app.

Does that seem reasonable? I am happy to pair on this if something is unclear.

Reply:
I agree about not having overly-specific components. What do you think about atoms like `SmallText`, `LargeText`, `MicroText`, etc. instead of `SecondaryText` and `TertiaryText` though (I got the idea from the article above)? These would replace `Paragraph` and `Header1` and the like.

I think having atoms like `SecondaryText` still leaves too much ambiguity, because it forces the consumer of these components to reason about page and content hierarchy before making a style decision. I find reasoning about hierarchy in this codebase difficult because a lot is changing and we haven't systematized the design at a high level yet.

What do you think?

Reply:
>. What do you think about atoms like SmallText, LargeText, MicroText, etc. instead of SecondaryText and TertiaryText 

I like that idea :+1:

We may also want to include `Title1`, `Title2`, and `Title3`, as shown in the article. And I think the `Title` atoms should replace our current `Header` atoms.

I am glad we are thinking more carefully about our design language. I think we can incrementally fix these atoms, instead of building upon a faulty design language. I also found this article helpful when thinking about a Design Language System: https://medium.com/thinking-design/what-is-a-design-language-really-cd1ef87be793

We can look at other companies' DLS's as a source of inspiration as well, like Trello: https://design.trello.com/style/typography

And here's another article I came across as well: https://uxdesign.cc/atomic-design-how-to-design-systems-of-components-ab41f24f260e

Reply:
I think I'll turn these `PanelLabel`s into `MediumText` components.

I think you're probably right we can do this incrementally, but I also bet we'll benefit from thinking through the whole design system up front. We don't have to get every detail in place though.

There's a Quip doc called `Design Language` where I'm trying to outline our design language.